curriculum/challenges/english/25-front-end-development/review-javascript-maps-sets-and-json/6723d027b02e4cc6ee5944da.md

```diff
@@ -9,7 +9,37 @@ dashedName: review-javascript-maps-sets-and-json
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## Sets in JavaScript
 
+- A Set is a built-in option for managing data collection. 
+- Sets ensure that each value in it appears only once, making it useful for eliminating duplicates from an array or handling collections of distinct values.
+- You can create a Set using the `Set()` constructor.
+- Sets can be manipulated using methods like `add()`, `delete()`, `has()`, and `clear()`.
```

Comment:
I haven't read the lecture script for this, but I would provide a short sentence for each method here